recipes/sensor.rb

```diff
@@ -0,0 +1,61 @@
+# install and run the metaflows sensor/collector
+include_recipe 'runit'
+
+if node['platform'] == 'centos'
```

Comment:
I need to write some documentation around this. The cookbook runs on the metaflows ami in aws, and that box is only CentOS. They don't support other platforms.